prow/plugins/cla/cla_test.go

```diff
@@ -140,6 +141,34 @@ func TestCLALabels(t *testing.T) {
 			addedLabels:   []string{fmt.Sprintf("/#3:%s", labels.ClaNo)},
 			removedLabels: []string{fmt.Sprintf("/#3:%s", labels.ClaYes)},
 		},
+		{
+			name:      "EasyCLA status failure removes \"cncf-cla: yes\" label",
+			context:   "EasyCLA",
+			state:     "failure",
```

Comment:
Since the plugin now supports multiple contexts, I think it's fair to ask the test cases be written to test multiple contexts (or that you add a new Test that supports this).

I would like to see test cases with multiple contexts having conflicting values, so we can confirm that it's last-context-wins in all cases

Reply:
e.g. I would like to see something like
```go
contexts:
  "cla/linuxfoundation": "success" 
  "EasyCLA": "failure"
```

and vice-versa